EKF/common.h

```diff
@@ -133,8 +161,12 @@ struct parameters {
 	float pos_noaid_noise = 10.0f;      // observation noise for non-aiding position fusion
 	float baro_noise = 3.0f;            // observation noise for barometric height fusion
 	float baro_innov_gate = 3.0f;       // barometric height innovation consistency gate size in standard deviations
+	float range_noise = 0.1f;
+	float flow_innov_gate = 3.0f;
+	float flow_noise = 0.001f;
```

Comment:
If this is a 1-sigma noise and not a variance, then it is far too small for a default parameter setting and could drive the filter unstable or cause rejection of flow measurements.